data/campaigns/The_South_Guard/scenarios/06a_Tidings_Good_and_Ill.cfg

```diff
@@ -365,7 +365,7 @@
             message= _ "Ho! Me see elves. Elves tasty? Me take elves to eat."
         [/message]
         [message]
-            race=elf
+            speaker=second_unit
```

Comment:
Or you can simply add a #po comment telling translators to consider that the speaker can be of either gender, so they can choose a gender-ambiguous wording. However, considering that unspecified units talk in campaigns all the time, I'd think translators wouldn't have trouble figuring that out on their own in the first place.